public_html/common.php

```diff
@@ -67,10 +68,12 @@ function load_wiki_name($query_array) {
 	  'wiktionary' => 'wiktionary',
 	);
 	$short_project = $project_map[$project];
-	if ( $project === 'wikidata' ) {
-		$wiki_name = 'wikidatawiki';
```

Comment:
I made the changes to be merged later [here](https://github.com/WikiEducationFoundation/WikiEduDashboard/pull/1262/files#diff-c715d651a0bad079aed15d6c23b43ed0R168), along with the changes for sources wiki and wikimedia incubator earlier, as special cases for database names. Do let me know if I need to revert the 'wikidata' condition here for the time being.  Thanks!